jsapp/js/components/modalForms/projectSettings.es6

```diff
@@ -711,17 +712,11 @@ class ProjectSettings extends React.Component {
           </bem.FormModal__item>
 
           <bem.FormModal__item m='metadata-share'>
-            <input
-              type='checkbox'
-              id='share-metadata'
+            <Checkbox
               checked={this.state['share-metadata']}
               onChange={this.onShareMetadataChange}
+              label={t('Help KoboToolbox improve this product by sharing the sector and country where this project will be deployed.') + ' ' + t('All the information is submitted anonymously, and will not include the project name or description listed above.')}
```

Comment:
Even the old lines were too long to fit in GitHub's code view without scrolling horizontally, but if you're going to keep `') + ' ' + t('`, why not split it across two lines?